app/javascript/components/components/Header.js

```diff
@@ -1,25 +1,62 @@
-import React, { Component } from 'react'
-import { NavLink } from 'react-router-dom'
+import { NavLink } from "react-router-dom";
+import React, { useState } from "react";
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+} from "reactstrap";
 
+const Header = (props) => {
+  const [collapsed, setCollapsed] = useState(true);
 
-class Header extends Component{
-    render(){
-        const {
-            logged_in,
-            sign_in_route,
-            sign_out_route
-        } = this.props
-        return(
-            <header id = "header">
-                <h3>Header</h3>
-                { logged_in && <a href={ sign_out_route }>Sign Out</a> }
-                {!logged_in && <a href={ sign_in_route }>Sign In</a>}
-                <br></br>
-                {this.props.logged_in && <NavLink to="/userrecipes"> Go to your Recipes Page!</NavLink>}
-            </header>
-        
-        )
-    }
-}
+  const toggleNavbar = () => setCollapsed(!collapsed);
 
-export default Header
+  //class Header extends Component {
+  // render() {
+  const { logged_in, sign_in_route, sign_out_route } = props;
+  return (
+    <div>
+      <Navbar color="faded" light>
+        <NavbarBrand href="/" className="header-menu">
+          Header
+        </NavbarBrand>
+        <NavbarToggler onClick={toggleNavbar} className="hamburger-menu" />
+        <Collapse isOpen={!collapsed} navbar>
+          <Nav navbar>
+            <NavItem>
+              {logged_in && <a href={sign_out_route}>Sign Out</a>}
+              {!logged_in && <a href={sign_in_route}>Sign In</a>}
+            </NavItem>
+            <NavItem>
+              <NavLink to="/components/pages/Home">Home</NavLink>
+            </NavItem>
+            <NavItem>
+              <NavLink to="/components/pages/RecipeIndex">
```

Comment:
I believe the path for the `to` prop should be using routes declared in `App.js`

Reply:
Got it